catchup/catchpointService.go

```diff
@@ -300,6 +305,13 @@ func (cs *CatchpointCatchupService) processStageLedgerDownload() (err error) {
 	return nil
 }
 
+// updateVerifiedAccounts update the user's statistics for the given verified accounts
+func (cs *CatchpointCatchupService) updateVerifiedAccounts(verifiedAccounts uint64) {
```

Comment:
true; the only downside is that this sync/atomic would need to be used on "both" ends. Since we already had that data structure, I figured that I'll reuse the existing mechanism.

Yes, using atomic would definitely be my choice as well if it was "just" this variable.